Procfile

```diff
@@ -1 +1 @@
-web: newrelic-admin run-program gunicorn __init__:app
+web: npm run build && newrelic-admin run-program gunicorn __init__:app
```

Comment:
No. This patch uses both the python and node buildpacks, so CF autodetects `package.json` and runs `npm install`.